docs/source/udf/advanced.rst

```diff
@@ -579,20 +579,20 @@ Threading
 ---------
 
 By default, LiberTEM uses multiprocessing with one process per CPU core for
-offline processing. In that scenario, UDFs should only use a single thread to
-avoid oversubscription.
+offline processing :class:`~libertem.executor.dask.DaskJobExecutor`. In that
+scenario, UDFs should only use a single thread to avoid oversubscription.
```

Comment:
I think the sentence is missing something, like "[...] for offline processing _using_ `DaskJobExecutor`" or similar?

Reply:
I've made that change, thanks!